wgsl/index.bs

```diff
@@ -1039,15 +1039,42 @@ switch_stmt
   : SWITCH paren_rhs_stmt BRACKET_LEFT switch_body+ BRACKET_RIGHT
 
 switch_body
-  : CASE const_literal COLON BRACKET_LEFT case_body BRACKET_RIGHT
+  : CASE case_selectors COLON BRACKET_LEFT case_body BRACKET_RIGHT
   | DEFAULT COLON BRACKET_LEFT case_body BRACKET_RIGHT
 
+case_selectors
+  : const_literal (COMMA const_literal)*
+
 case_body
   :
   | statement case_body
   | FALLTHROUGH SEMICOLON
 </pre>
 
+A switch statement transfers control to one of a set of case clauses, or to a default clause,
+depending the evaluation of a selector expression of a scalar integer type.
+
+If the selector value equals a value in a case selector list, then control is transferred to
+the body of that case clause.
+If the selector value does not equal any of the case selector values, then control is
+transferred to the `default` clause.
+
+TODO(dneto): Is the default clause required? That would avoid undefined behaviour.
+
+A literal value must not appear more than once in the case selectors for a switch statement.
```

Comment:
Does the type of the literal value have to match the type of the switch conditional? I think yes, but it should probably be called out here.

Reply:
Added a sentence for this.